README.md

```diff
@@ -400,7 +400,7 @@ Note that if you are using your own application loader then you should ensure th
 ```java
 class MyCustomApplicationLoader extends ApplicationLoader {
   def load(context: ApplicationLoader.Context): Application = {
-    val conductRConfig = Configuration(AkkaEnv.asConfig) ++ Configuration(PlayEnv.asConfig) ++ Configuration(LagomEnv.asConfig)
+    val conductRConfig = Configuration(AkkaEnv.asConfig) ++ Configuration(PlayEnv.asConfig)
```

Comment:
The README section for Lagom is a bit outdated (doesn't reflect the Java/Scala split) but I thought I would fix that in a separate pull request after this is merged.